packages/gestalt/src/TableOfContents.test.js

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+import { create } from 'react-test-renderer';
+import TableOfContents from './TableOfContents.js';
+
+describe('TableOfContents', () => {
+  it('renders', () => {
+    const tree = create(
+      <TableOfContents title="Title" items={[{ label: 'Label', href: '#' }]} />,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders an accessibility label', () => {
```

Comment:
I think more snapshots are needed here. They're probably going to be super similar to the examples you'll write